requirements_all_ds.txt

```diff
@@ -30,3 +30,11 @@ snowflake_connector_python==1.6.10
 phoenixdb==0.7
 # certifi is needed to support MongoDB and SSL:
 certifi
+<<<<<<< HEAD
```

Comment:
Seems you missed some conflicts :wink:

Reply:
Damn I just woke up, didn't get enough coffee I guess... ;)